guide_categories.json

```diff
@@ -60,11 +60,11 @@
         [
             {
                 "subcategory_name": "Containerize",
-                "guides": ["containerize", "kubernetes-microprofile-config", "istio-intro", "kubernetes-intro"]
+                "guides": ["containerize", "kubernetes-intro", "istio-intro", "guide-spring-boot"]
```

Comment:
You don't need the "guide-" in "guide-spring-boot". Just "spring-boot" should work